src/styles/styles.js

```diff
@@ -2860,6 +2860,21 @@ const styles = {
         fontSize: variables.fontSizeNormal,
         maxWidth: 240,
     },
+
+    welcomeTextHeader: {
+        fontSize: variables.fontSizeHero,
+    },
+
+    emptyStateBackgroundContainer: {
+        width: '100%',
+        position: 'absolute',
+    },
+
+    emptyStateBackgroundImage: {
+        width: '100%',
+        height: '80%',
+        bottom: '-40%',
```

Comment:
Real quick, what's the reasoning behind these %?

Reply:
Sorta same as above. It's what made it work, haha. Height and width need to be defined or else the image doesnt' show up. The width needs to be 100% or greater or else you see the edges on desktop, and the height was just the value that made it the right size for the mocks. The `bottom` moves it up to the right level to match.

Reply:
Oh interesting, I didn't know that about FlatList. I didn't notice any visible issues when scrolling through longer chats, but would be good to test more. As for the percentages, it was basically just the values that made things work. I can try to find pixel values instead, and then can add them to the CONST file.

Reply:
Update on this - I use the percentages here vs. pixel sizes since the size of the container changes depending on whether it's wide screen or not. Pixel sizes would make it go weird on one or the other.